acme/setup.py

```diff
@@ -22,15 +22,14 @@
 ]
 
 # env markers in extras_require cause problems with older pip: #517
+# Keep in sync with conditional_requirements.py.
 if sys.version_info < (2, 7):
     install_requires.extend([
         # only some distros recognize stdlib argparse as already satisfying
         'argparse',
-        'mock<1.1.0',
```

Comment:
@kuba absolutely yes!  We're not going to land the mock removal in a hurry and without your agreement.  Regarding your questions:

1. Obviously we can't modify the `setup.py` of third party dependencies unless we fork of vendorise those projects, though in some cases (ConfigArgParse) we've already been tempted to do that, and it's not totally out of the question.  We'd always prefer to work with those projects to keep their dependency sets minimal if possible.

2. How would you feel about a separate PR against `acme` which (1) removes mock from the main dependency set; (2) raises a planned exception if the mock-dependent test code is called without mock installed; (3) documents this behaviour as part of the API?